dev/setup-clowder.sh

```diff
@@ -181,4 +181,7 @@ END_TIME=`date +%s`
 DEPLOYMENT_TIME=`expr $END_TIME - $START_TIME`
 print_message "Deployment of strimzi kafka, host-inventory, and xjoin-operator took $DEPLOYMENT_TIME seconds"
 
+# Run the operator code to create "xjoin.inventory.hosts"
+make run ENABLE_WEBHOOKS=false &
```

Comment:
please remove this. This script is to setup the dependencies. Running the operator is a separate operation so it can be ran from an IDE or elsewhere.